app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,32 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @comment = current_user.comments.build(comment_params)
+    @comment.save
+    respond_to do |format|
```

Comment:
to avoid this duplication in all actions you can use `default: { format: :json }` in routes
please investigate and do it